service/src/main/java/uk/gov/hmcts/reform/fpl/service/RepresentativeService.java

```diff
@@ -211,4 +214,29 @@ private void associatedRepresentativeWithParty(CaseData caseData, Element<Repres
                 return Optional.empty();
         }
     }
+
+    public List<Element<Representative>> getRepresentativePartiesToNotify(CallbackRequest request) {
+        CaseData caseData = mapper.convertValue(request.getCaseDetails().getData(), CaseData.class);
+        CaseData caseDataBefore = mapper.convertValue(request.getCaseDetailsBefore().getData(), CaseData.class);
+
+        if (isNotEmpty(caseDataBefore.getRepresentatives())) {
+            if (!caseDataBefore.getRepresentatives().containsAll(caseData.getRepresentatives())) {
```

Comment:
why do we need this condition? Why would we not run `getChangedRepresentatives` if `caseData.getRepresentatives()` included a new representative for example?

Reply:
good spot changed and working :)